src/tokens/hooks.ts

```diff
@@ -61,6 +67,51 @@ export function useTokenPricesAreStale(networkIds: NetworkId[]) {
   }
 }
 
+export function useSendableTokens() {
+  // TODO: get this from statsig
+  const networkIdsForSend = [networkConfig.defaultNetworkId]
+  const tokens = useSelector(tokensListSelectorWrapper(networkIdsForSend))
+  return tokens.filter((tokenInfo) => tokenInfo.balance.gt(TOKEN_MIN_AMOUNT))
+}
+
+export function useSwappableTokens() {
+  const appVersion = DeviceInfo.getVersion()
+  // TODO: get this from statsig
+  const networkIdsForSwap = [networkConfig.defaultNetworkId]
+  const tokens = useSelector(tokensListSelectorWrapper(networkIdsForSwap))
+  return tokens
+    .filter(
+      (tokenInfo) =>
+        tokenInfo.isSwappable ||
+        (tokenInfo.minimumAppVersionToSwap &&
+          !isVersionBelowMinimum(appVersion, tokenInfo.minimumAppVersionToSwap))
+    )
+    .sort(tokenCompareByUsdBalanceThenByName)
+}
+
+function isCiCoToken(tokenSymbol: string): tokenSymbol is CiCoCurrency {
```

Comment:
what do you think about abandoning the "CiCo" casing pattern? seems like proper camelCase would just be CICO

Reply:
I ended up changing to Cico to be consistent with the feature gate `showCico`